tests/test_stubs.py

```diff
@@ -57,7 +57,10 @@ def test_fail(path, py2_arg):
             continue
 
         lineno = int(error_line.split(":")[1])
-        errors[lineno] = error_line
+        if lineno not in errors:
+            errors[lineno] = error_line
+        else:
+            errors[lineno] += '\n' + error_line
```

Comment:
Change required to support matching within multi-output failures (e.g. when incorrectly using overloads), as previously this would only allow us to match against the last line of the error.